docs/community/contributing-emulators.md

```diff
@@ -0,0 +1,69 @@
+# Contributing emulators
+
+This guide explains how to contribute new emulator models to `AutoEmulate`.
+
+## Emulator structure
+
+All emulators in AutoEmulate are implemented as `scikit-learn` estimators, making them compatible with scikit-learn's cross-validation, grid-search, and pipeline functionality. Have a look at the [scikit-learn estimator developer guide](https://scikit-learn.org/1.5/developers/develop.html#rolling-your-own-estimator) for more details on how to implement a new estimator.
+
```

Comment:
I suggest we add a note on the limitations regarding time-series data, since this has come up so often in conversations. I am not sure where the best place for this is, we might instead have a discussion post around this, but a brief note here would be helpful. Something along the lines of :

```suggestion

**Note**: AutoEmulate is designed primarily for static data analysis, leveraging its integration with scikit-learn. If you are contributing an emulator for time-series data, keep in mind that it may not perform optimally without additional handling of temporal dependencies, particularly during cross-validation and evaluation.
```